datasets/goes/goes-glm/dataset.yaml

```diff
@@ -0,0 +1,36 @@
+id: goes_glm
+image: ${{ args.registry }}/pctasks-task-base:latest
+
+args:
+- registry
+
+code:
+  src: ${{ local.path(./goes_glm.py) }}
+  requirements: ${{ local.path(./requirements.txt) }}
+
+environment:
+  AZURE_TENANT_ID: ${{ secrets.task-tenant-id }}
+  AZURE_CLIENT_ID: ${{ secrets.task-client-id }}
+  AZURE_CLIENT_SECRET: ${{ secrets.task-client-secret }}
+
+collections:
+  - id: goes-glm
+    template: ${{ local.path(./collection) }}
+    class: goes_glm:GoesGlmCollection
+    asset_storage:
+      - uri: blob://goeseuwest/noaa-goes16/
+        token: ${{ pc.get_token(goeseuwest, noaa-goes16) }}
+        chunks:
+          options:
+            name_starts_with: GLM-L2-LCFA/2022/190/
+            ends_with: .nc
+            chunk_length: 180
+      - uri: blob://goeseuwest/noaa-goes17/
+        token: ${{ pc.get_token(goeseuwest, noaa-goes17) }}
+        chunks:
+          options:
+            name_starts_with: GLM-L2-LCFA/2022/190/
```

Comment:
Same question about removing the `2022/190`.

Reply:
Fixed.